packages/lit-html/src/directives/live.ts

```diff
@@ -14,6 +14,24 @@ import {
 } from '../directive.js';
 import {isSingleExpression, setCommittedValue} from '../directive-helpers.js';
 
+type ValidPartType = 1 | 3 | 4;
+type Element = HTMLElement & {[key: string]: unknown};
```

Comment:
I don't think we should override the global `Element` type, and this type could remove a lot of type-checking in the whole file. We can cast at the access site.